connectors/es/sink.py

```diff
@@ -365,6 +374,14 @@ async def run(self, generator, job_type):
         except asyncio.CancelledError:
             self._logger.info("Task is canceled, stop Extractor...")
             raise
+        except asyncio.QueueFull as e:
+            self._logger.critical("Throttled by Elasticsearch", exc_info=True)
+            # We clear the queue as we could not actually ingest anything.
+            # After that we indicate that we've encountered an error
+            self.queue.clear()
+            await self.put_doc("FETCH_ERROR")
+            msg = "Connector was unable to ingest data into overloaded Elasticsearch. Make sure Elasticsearch instance is healthy, has enough resources and content index is healthy."
```

Comment:
worth still sharing how long the queue was full for? Like `"Connector was unable to ingest data into Elasticsearch for 10 minutes. Elasticsearch may be overloaded. Make sure ...` ?

Reply:
There's no access to the number of minutes here, but it gave me the idea to mention the configuration setting that's responsible for this: 

> Connector was unable to ingest data into overloaded Elasticsearch. Make sure Elasticsearch instance is healthy, has enough resources, content index is healthy or increase \"elasticsearch.bulk.queue_refresh_timeout\" setting.